src/app_service/service/chat/service.nim

```diff
@@ -422,26 +422,26 @@ QtObject:
       error "error: ", errDesription
       return
 
-  proc addGroupMembers*(self: Service, communityID: string, chatID: string, members: seq[string]) =
+  proc addGroupMembers*(self: Service, chatID: string, members: seq[string]) =
     try:
-      let response = status_group_chat.addMembers(communityID, chatId, members)
+      let response = status_group_chat.addMembers("", chatId, members)
```

Comment:
Those functions were abstracted so that they could work for bot communities and group chats. You should keep the arg here and in the controller send `sectionId` or `""` depending on if it's a community.

I see that for now, those function are only called in the specific group chat context, but we'll want to call the same function for both at some point.

Reply:
@jrainville I removed `communityID` only from the procs which are "chat section only" features (or personal chats group) like create 1:1 chat, create gorup chat, edit group chat name and so. Exposing `communityID` as param of those method to qml and setting it for `""` string is somehow redundant and maybe confusing (for example dev may search for community id on the qml side from the component where it is not easily accessible just to satisfy params list of that method).

Once we want to introduce group chats for communities we will add it then, but that may never happen for example.